fastly/s3.go

```diff
@@ -9,12 +9,24 @@ import (
 
 type S3Redundancy string
 type S3ServerSideEncryption string
+type S3AccessControlList string
 
 const (
-	S3RedundancyStandard      S3Redundancy           = "standard"
-	S3RedundancyReduced       S3Redundancy           = "reduced_redundancy"
+	S3RedundancyStandard   S3Redundancy = "standard"
+	S3RedundancyReduced    S3Redundancy = "reduced_redundancy"
+	S3RedundancyOneZoneIA  S3Redundancy = "onezone_ia"
+	S3RedundancyStandardIA S3Redundancy = "standard_ia"
```

Comment:
There looks to be more values defined in https://docs.aws.amazon.com/AmazonS3/latest/API/API_CreateMultipartUpload.html#API_CreateMultipartUpload_RequestParameters (under `x-amz-storage-class`).

I presume those extra values were omitted because the Fastly UI doesn't make reference to them (which makes sense).

Reply:
Yes similar to the ACL values, the main reason was that they weren't specified in the UI.